modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/util/index.js

```diff
@@ -224,7 +224,7 @@ export function parseOptions(jsonString) {
 	}
 
 	return Array.isArray(options)
-		? options.map(({value}) => `${value}`).join(', ')
+		? options.map(({name}) => `${name}`).join(', ')
```

Comment:
This doesn't look right, what if I have a text option? I see only the text option name without the value that I set?

Reply:
I cannot alter the value implementation because it is too embedded in our code.

Reply:
If you try your reproduction steps again, but change the name of the product option values to something more visible like "6name", etc. You will see the correct results of the PR.

Reply:
<img width="885" alt="Screenshot 2023-10-04 at 11 53 49" src="https://github.com/liferay-commerce/liferay-portal/assets/580733/4b601594-75c8-4046-a0be-1c34a9e49214">

I got the point of showing the optionvalue name, and that's fine, but as I thought, this fix is not working for all of the option type, in the screenshot it's failing on text type. Can you please verify what happens with all of the option type and also please sync your solution with COMMERCE-9599, because it changes quite a bit the options in the cart item component.

Thank you!

Reply:
@andrea-ale-sbarra ,

I should have a solution soon. You are right with your testing. There are lot of edge cases which I should be able to take care of.

Sincerely,
Brian I. Kim

Reply:
See updated commits @andrea-ale-sbarra , tested with COMMERCE-9599 included.

Reply:
See https://liferay.atlassian.net/browse/COMMERCE-12528 for limitation on Boolean.